target/classes/plugin.yml

```diff
@@ -5,7 +5,7 @@ softdepend: [mcMMO]
 authors: [Big_Scary, ShaneBee, DmitryRendov]
 description: GriefPrevention Add-On to set flags in claims.
 website: https://www.spigotmc.org/resources/griefprevention-flags.55773/
-version: '5.6.4'
+version: '5.7.2'
```

Comment:
Can you please also revert this change. I'm not sure that it will really matter, but regardless this didn't need to be part of this PR